rpc_jobs/rpc_aio.yml

```diff
@@ -335,6 +343,11 @@
                   holland.holland()
                 }} else if (env.STAGES.contains("Leapfrog Upgrade")) {{
                   deploy.upgrade_leapfrog(environment_vars: environment_vars)
+                  maas.deploy()
+                  maas.verify()
+                  tempest.tempest()
+                  kibana.kibana(env.KIBANA_SELENIUM_BRANCH)
+                  holland.holland()
```

Comment:
I think it's worth removing. If we test holland in our env, it means that we have to include the support role. 
However, here we are signing a blank cheque by doing so: We can't do anything about the support role, and we don't audit or even gate test it as standalone ourselves.
It means it may be deleting the complete environment, or any other thing we don't want (installing apt sources, conflictual pip packages...?), and we'll have to deal with consequences: our job will never pass in that events. 

I'd rather reduce test coverage than blindly running code.